server/budgettracker.web/Controllers/AuthenticationApiController.cs

```diff
@@ -41,5 +41,11 @@ public ApiResponse Authenticate(ApiRequest request)
         {
             return _authApi.AuthenticateUser(request);
         }
+
+        [HttpPost("delete")]
+        public ApiResponse DeleteUser(ApiRequest request)
```

Comment:
We should be returning a Task<IActionResults> instead I think. Since we should be making this call async if we are making DB calls. Then for return, you just do return Ok(await *method*);